srcpkgs/brother-brlaser/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'brother-brlaser'
+pkgname=brother-brlaser
+version=4
+revision=1
+create_wrksrc=yes
```

Comment:
why create_wrksrc?. `wrksrc="brlaser-${version}"` should do the job.

Reply:
Just checked it locally, it does work, you need to change `build_wrksrc` to `wrksrc`

Reply:
Thanks, changed.